docs/_includes/field.html

```diff
@@ -961,4 +961,119 @@ <h3 id="field-fluid">Fluid Field</h3>
 </div>
         {% endhighlight %}
 
+    <h3 id="field-character-count">Field Character Count</h3>
+    <p>A field may have a character count section typically under the input and aligned to the right.</p>
+
+    This variant adds a top-level container that houses both the field description and the character count, though field description or error is not necessary.
+
+    <p>In order to be accessible, the character count should have a clipped element to describe what it is. At the same time, aria-describedby should be added to the input element that points to the character count element. Finally, <span class="highlight">aria-live</span> should be off by default, but if it reaches 10 characters or over the current max characters, then it should be changed to polite. Once it goes back down below 10 characters the max, it should go back to off.</p>
+
+    <h4>Field with Character Count Only</h4>
+    <p>Character character can be added by utlising CSS grid via the <span class="highlight">field__description--group</span> modifier and the addition of the relevant markup. By default if there is only one element in the group, it will align to the right</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="field">
+                <label class="field__label field__label--stacked" for="field-character-2-input">Field 2</label>
+                <span class="field__control textbox">
+                    <input type="text" maxlength="140" aria-describedby="field-character-2" class="textbox__control" id="field-character-2-input" aria-live="off"  data-field-count="field-character-2"/>
+                </span>
+                <div class="field__description field__description--group">
+                    <div id="field-character-2">
+                        0 of 140
+                        <span class="clipped"> charaters</span>
+                    </div>
+                </span>
```

Comment:
Small typo, causing some weird problems with the mismatched closing tags
```suggestion
                </div>
```

Reply:
Done